deploy/olm-catalog/ibm-block-csi-operator-community/1.7.0/ibm-block-csi-operator.v1.7.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,681 @@
+      mediatype: image/png
+  labels:
+    alm-owner-ibmblockcsi: ibmblockcsioperator
+    operated-by: ibmblockcsioperator
+  selector:
+    matchLabels:
+      alm-owner-ibmblockcsi: ibmblockcsioperator
+      operated-by: ibmblockcsioperator
+  installModes:
+    - type: OwnNamespace
+      supported: true
+    - type: SingleNamespace
+      supported: true
+    - type: MultiNamespace
+      supported: false
+    - type: AllNamespaces
+      supported: false
+  customresourcedefinitions:
+    owned:
+      - name: ibmblockcsis.csi.ibm.com
+        version: v1
+        group: csi.ibm.com
+        kind: IBMBlockCSI
+        displayName: "IBM block storage CSI driver"
+        description: "Represents an block storage CSI driver"
```

Comment:
could you [fix this](https://github.com/IBM/ibm-block-csi-operator/pull/141#discussion_r631224620) while you're at it?

Reply:
Done